pgjdbc/src/main/java/org/postgresql/core/Parser.java

```diff
@@ -835,6 +837,28 @@ public static boolean parseEndKeyword(final char[] query, int offset) {
         && (query[offset + 2] | 32) == 'd';
   }
 
+  /**
+   * Parse string to check presence of AS keyword regardless of case.
```

Comment:
Shouldn't this comment be check the presence of ROLLBACK ?

Reply:
Yes, I trust Copilot too much...